src/main/java/duke/DukeOperations.java

```diff
@@ -0,0 +1,92 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * Class containing all the various Duke program methods.
+ */
+public class DukeOperations {
+
+
+
+    /**
+    * method that returns exit message.
+    *
+    */
+    public void exit() {
+        System.out.println("_________________________________________________________________________");
+        System.out.println("        Bye. Hope to see you again soon!");
+        System.out.println("_________________________________________________________________________");
+    }
+
+    /**
+    * Displays list of all tasks adeed/
+    * @param array contains arraylist of all the tasks
+    * @param counter integer value of the number of tasks in the array so far.
+    */
+    public void displayList(ArrayList<Task> array, int counter) {
+        System.out.println("_________________________________________________________________________");
+        for (int i = 0; i < counter; i++) {
+            if (array.get(i) != null) {
+                int j = i + 1;
+                System.out.println(j + ". " + array.get(i).toString());
+            }
+        }
+        System.out.println("_________________________________________________________________________");
+    }
+
+    /**
+    * Marks as a task as done and displays output messages.
+    * @param array arraylist of all the tasks so far.
+    * @param number integer value of the task to be marked as done.
+    */
+    public void mark(ArrayList<Task> array, int number) {
+        array.get(number-1).markAsDone();
+        System.out.println("_________________________________________________________________________");
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(array.get(number-1).toString());
+        System.out.println("_________________________________________________________________________");
+        Storage.clearFile();
+        Storage.writeToFile(array,array.size());
+    }
+
+    /**
+     * Unmarks as a task as done and displays output messages.
+     *
+     * @param array arraylist of all the tasks so far.
+     * @param number integer value of the task to be unmarked as not done.
+     * */
+     public void unmark(ArrayList<Task>array, int number) {
```

Comment:
For method naming, perhaps you can use unMark instead of unmark. I also noticed similar issues in other methods.